libr/anal/p/anal_sbpf.c

```diff
@@ -70,23 +75,30 @@ static bool sbpf_check_string_pointer(RAnal *anal, ut64 ptr_addr, ut64 data_star
 	// Get size from offset 8
 	ut64 size = r_read_le64 (struct_buf + 8);
 
+	R_LOG_DEBUG ("  Checking potential string pointer: str_ptr=0x%"PFMT64x", size=0x%"PFMT64x, str_ptr, size);
+
 	// Validate size is reasonable (but not too small, and not a typical pointer value)
 	if (size == 0 || size > 0x100) {
+		R_LOG_DEBUG ("  Rejected: size out of range (0 or > 0x100)");
 		return false;
 	}
 
 	// Check if string pointer is in data segment
 	if (str_ptr < data_start || str_ptr >= data_end) {
+		R_LOG_DEBUG ("  Rejected: str_ptr 0x%"PFMT64x" not in data segment [0x%"PFMT64x" - 0x%"PFMT64x")",
+			str_ptr, data_start, data_end);
 		return false;
 	}
 
 	// Try to read the actual string to verify it's printable
 	char sample[0x100];
 	ut32 sample_size = (size < 0x100) ? size : 0x100;
 	if (!anal->iob.read_at (anal->iob.io, str_ptr, (ut8 *)sample, sample_size)) {
+		R_LOG_DEBUG ("  Rejected: failed to read string at 0x%"PFMT64x, str_ptr);
```

Comment:
no heading spaces. do the same for the rest of logdebugs

```suggestion
		R_LOG_DEBUG ("Rejected: failed to read string at 0x%"PFMT64x, str_ptr);
```